x-pack/plugins/lens/public/visualizations/xy/legend_stats_telemetry_helpers.ts

```diff
@@ -10,7 +10,7 @@ import { XYLegendValue } from '@kbn/visualizations-plugin/common';
 import { nonNullable } from '../../utils';
 import { shouldDisplayTable } from '../../shared_components/legend/legend_settings_popover';
 
-const LEGEND_STATS_PREFIX = 'lens_legend_stats';
+const LEGEND_STATS_PREFIX = 'legend_stats';
```

Comment:
It's great that you also found out about this one! Let's ping @teresaalvarezsoler and update the spreadsheet with both keys used. We should probably also mark in the spreadsheet the time when the new telemetry key was merged/released so we can use both or just one key depending on the queried time.
Or we can update the events directly in our telemetry by renaming the values

Reply:
Hey @markov00 thanks. I edited the spreadsheet yesterday with the new and old names and a comment.